src/components/Comments.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import { getCommentForSingleReview } from "../axios/apiQueries";
+
+const Comments = ({ review_id }) => {
+  const [commentsList, setCommentsList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getCommentForSingleReview(review_id).then((comments) => {
+      setCommentsList(comments);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  return isLoading ? (
+    <h1>Loading...</h1>
+  ) : (
+    <div>
+      {commentsList.length === 0 ? (
+        <h2>No comments yet</h2>
```

Comment:
Nice! A good distinction between loading and informing the user the list is blank correctly